app/graphql/types/search_export_type.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Types
+  class SearchExportType < Types::BaseObject
+    # it makes me sad that we're not using relay-style ids everywhere
+    field :id, Integer, null: false
```

Comment:
ID obfuscation is a good practice in GraphQL. Check out [Production Ready GraphQL](https://productionreadygraphql.com/) for good practices around schema design.